releng/org.eclipse.n4js.targetplatform/org.eclipse.n4js.targetplatform.target

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde version="3.8"?>
-<target name="Generated target-platform N4JS:N4JS_Neon.2.GH-18" sequenceNumber="135">
+<target name="Generated target-platform N4JS:N4JS_Oxygen.2.GH-369" sequenceNumber="141">
```

Comment:
as with Oomph setup, change to `Generated target-platform N4JS:N4JS_Oxygen" sequenceNumber="141`